### DISCUSSION TITLE
question setup reset passworld jellyfin

### DISCUSSION THREAD

**Discussion by NicolasB73** [hrfee/jfa-go#253] [General]
Hello I installed jellyfin on docker and I can't get a setup to reset the passwords

**Comment by hrfee** [comment]
Have you mounted Jellyfin's config directory in jfa-go's container? Put the directory you have mounted to the Jellyfin container as `/config` as the `/jf` mount on your jfa-go container, then Enable password resets in jfa-go.

**Reply by InterN0te** [reply]
Are they an extension to edit reset password page and message ?
To easily allow reset over WAN and display the right instruction
